packages/lib-content/src/screens/About/About.js

```diff
@@ -70,6 +70,8 @@ function AboutPage() {
         src='https://widget.freshworks.com/widgets/44000004375.js'
         async
         defer
+        crossOrigin='anonymous'
+        integrity='sha384-GpJCLE7VNj6jp2jskwQ5wnbyfRLB4jsiVzyCg1pJuMBnAqjylfzWNhQu3yKQeidZ'
```

Comment:
This basically verifies that the content of the URL `https://widget.freshworks.com/widgets/44000004375.js` hasn't been modified from the version that was used to generate the integrity hash. If the script doesn't match, it won't run.